tasks/bump.js

```diff
@@ -46,7 +46,7 @@ module.exports = function(grunt) {
     var globalVersion; // when bumping multiple files
     var gitVersion;    // when bumping using `git describe`
 
-    var VERSION_REGEXP = opts.regExp || new RegExp(
+    var VERSION_REGEXP = (typeof(opts.regExp) == 'function') ? opts.regExp() : opts.regExp || new RegExp(
```

Comment:
if this is a function, then it should pass `opts.prereleaseName` so the function can use it